mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -23,42 +25,61 @@ class ExperimentPage extends Component {
     dispatchSearchRuns: PropTypes.func.isRequired,
   };
 
-  state = {
-    paramKeyFilter: new KeyFilter(),
-    metricKeyFilter: new KeyFilter(),
+  static defaultState = {
+    paramKeyFilterString: "",
+    metricKeyFilterString: "",
     getExperimentRequestId: getUUID(),
```

Comment:
I am not super familiar with JavaScript declarations here, but does defining this as static as opposed to part of the inline Component have any implications as to how often this `getExperimentRequestId` is instantiated with an ew UUID? For example, is it possible that in the previous code, every time I load the ExperimentPage I got a new UUID, and now every time I load the page (via a hash link, not a page reload) I may initialize to the same UUID?